src/App.js

```diff
@@ -25,7 +26,9 @@ function App() {
           }
         />
         <Route path="/" element={<Home />}/>
-        <Route path="/ProductListing" element={<ProductListing />}/>       
+        <Route path="/ProductListing" element={<ProductListing />}/>    
+
+        <Route path="/Cart" element={<Cart/>}/>  
```

Comment:
Suggestion : avoid unnecessary spacing , use Prettier Extension in VS Code. It will automatically format the code on Save.